FlowCrypt/Resources/en.lproj/Localizable.strings

```diff
@@ -83,6 +85,10 @@
 "compose_recipient" = "Add Recipient";
 "compose_subject" = "Subject";
 "compose_enable_search" = "To enable searching contacts, please Log out and set up FlowCrypt again";
+"compose_enable_contacts_search" = "To enable contacts search, please allow access to your Google contacts on the next screen";
```

Comment:
`allow access` sounds scary (allow to whom, where will it end up?)

Let's be more specific: `To enable contacts search, please allow this device to access Google Contacts.`

Reply:
also ` "compose_enable_search" = "To enable searching contacts, please Log out and set up FlowCrypt again"; ` is outdated. Is there a way to show warnings on unused strings?

Reply:
I found script which shows all unused localized strings, can just remove them from `Localizable.strings`

<img width="682" alt="Screenshot 2021-11-19 at 22 22 45" src="https://user-images.githubusercontent.com/1062093/142687103-98047260-1c5d-45e3-99e1-18b77238a0f4.png">